### DISCUSSION TITLE
Welcome to Vanilla OS Discussions!

### DISCUSSION THREAD

**Discussion by kbdharun** [Vanilla-OS/live-iso#342] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LeatherMolly** [comment]
Hi there,
I see no discussions here, maybe you are all on discord. But I can't connect to discord because some sort of safety check says that from my IP is coming too much I don't know what...so I'll try my best here.

I use VanillaOS as daily driver since 2026. I like it but ok gnome is a bit annoying, because it hides so many I am used to tune myself, in the LinuxMint days with Mate.

Ok, in the beginning I liked the stability. Since some time mostly every session freezes by firefox or other programs. I don't know where to check it, because everything is hidden...I mean I don't have a menu and can't look for programs to fix or to read logs.

So I ask myself, what the heck is doing this freezes? Often I see at the beginning a cursor, and then there is another that I can move.
Cool would be a gui where I could check the integrity of the system.

Do someone have some hints? Do some of you have similar experiences?

Why Apps are installed an have them with reral name and a second icon with "vso-pico"?

thx
Leathermolly

Is there a place where to get more support?
